src/index.js

```diff
@@ -415,6 +420,15 @@ const isNodeMatchingSelectorTabbable = function (options, node) {
   return true;
 };
 
+const isValidShadowCandidatesTabbable = function (node, nestedCandidates) {
```

Comment:
So here, `node` is expected to be the shadow host node, and `nestedCandidates` are all the candidate tabbable/focusable (already vetted as such) nodes that were found inside of it?

If so, could we clarify the expectation by renaming `node` to `shadowHostNode`?